lib/cli/command.js

```diff
@@ -89,9 +89,30 @@ export default class Command {
   allowExtraArgs = false;
 
   /**
+   * Returns the default and user specified environment variables.
+   *
+   * @method getEnvs
+   * @param project {Project} Used to set the environment of the project
+   */
+  getEnvs(project) {
```

Comment:
Not sure this belongs in the base command class. For example, port is specific to command in question (i.e. port won't really mean much to the test command). And the environment might not always want to be the project environment (i.e. I could see eventually differentiating between the build environment and runtime environment, i.e. build for test so that tests are included, but run with prod environment).